jax/_src/lax/other.py

```diff
@@ -117,16 +118,16 @@ def conv_general_dilated_patches(
 
 
 def conv_general_dilated_local(
-    lhs: jnp.ndarray,
-    rhs: jnp.ndarray,
+    lhs: jax.Array,
+    rhs: jax.Array,
```

Comment:
These should be `ArrayLike` for inputs, though this may require some code changes

Reply:
I fixed this particular example (seems fine) but left the others in this PR alone.

I also make jax.typing imported by default as part of `import jax`.